src/VDPDataStore.h

```diff
@@ -43,14 +47,20 @@ class VDPDataStore : public QObject, public SimpleHexRequestUser
 
 	void refresh1();
 	void refresh2();
+	void refresh3();
 
 private:
 	std::vector<uint8_t> vram;
 	size_t vramSize;
 
 	std::optional<std::string> debuggableNameVRAM; // VRAM debuggable name
 
+	bool registerLatchAvailable;
+	bool paletteLatchAvailable;
+	bool dataLatchAvailable;
```

Comment:
My personal preference is to initialize such simple variables directly in the .h file:
```
bool registerLatchAvailable = false;
bool paletteLatchAvailable = false;
bool dataLatchAvailable = false;
```